src/Symfony/Component/AssetMapper/CHANGELOG.md

```diff
@@ -9,13 +9,15 @@ CHANGELOG
  * Add "entrypoints" concept to the importmap
  * Always download packages locally instead of using a CDN
  * Allow relative path strings in the importmap
+ * Automatically set `_links` attribute for preload CSS files for WebLink integration
  * Add `PreAssetsCompileEvent` event when running `asset-map:compile`
  * Add support for importmap paths to use the Asset component (for subdirectories)
  * Removed the `importmap:export` command
  * Add a `importmap:install` command to download all missing downloaded packages
  * Allow specifying packages to update for the `importmap:update` command
  * Add a `importmap:audit` command to check for security vulnerability advisories in dependencies
  * Add a `importmap:outdated` command to check for outdated packages
+ * Automatically add Link preload headers for CSS files when using the WebLink component
```

Comment:
Looks like you've added the changelog twice.

Reply:
I just think it's a really great change 🫠